app/uk/gov/hmrc/gform/sharedmodel/VariadicFormData.scala

```diff
@@ -86,7 +87,7 @@ object SourceOrigin {
   def changeSourceToOutOfDate[M[_ <: SourceOrigin]](in: M[Current]): M[OutOfDate] = in.asInstanceOf[M[OutOfDate]]
 }
 
-case class VariadicFormData[S <: SourceOrigin](data: Map[ModelComponentId, VariadicValue]) {
+case class VariadicFormData[S <: SourceOrigin](data: collection.Map[ModelComponentId, VariadicValue]) {
```

Comment:
Can we keep this immutable?

Reply:
Hm, actually I see why this particular change is done like that. I'm fine with it 👍